src/index.html

```diff
@@ -9,59 +9,62 @@
     <meta name="keywords" content= "aurator">
     <meta name="author" content="Softwareberg">
 
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
     <title>Aurator</title>
 </head>
 <body class="app">
-    <div class="container">
+    <div class="container-fluid">
```

Comment:
When I was learning to use Bootstrap there was requirement to always use `row` and `col`

- [ ] do we need `row` and `col`